docs/estimators/alchemlyb.estimators.TI_GQ.rst

```diff
@@ -0,0 +1,12 @@
+.. _estimatators_TI_GQ:
+
+TI_GQ
+=====
+The :class:`~alchemlyb.estimators.TI_GQ` estimator is a simple implementation of `thermodynamic integration <https://en.wikipedia.org/wiki/Thermodynamic_integration>`_ that uses the `gaussian quadrature <https://en.wikipedia.org/wiki/Gaussian_quadrature>`_ for integrating the space between :math:`\left<\frac{dH}{d\lambda}\right>` values for each :math:`\lambda` sampled.
```

Comment:
I don't think you need the word `simple`. Using the word simple imply that there is a more complex implementation and we are not doing that.